terraform/install.sh

```diff
@@ -23,6 +23,8 @@ set -o errexit  # Exit on error
 SCRIPT_DIR=$(realpath $(dirname "$0"))
 cd $SCRIPT_DIR
 
+create_flag=false
```

Comment:
What is this? Please document the purpose of this variable

Reply:
It indicates whether user wants to create a project.

Reply:
add a comment